include/hpp/core/problem-solver.hh

```diff
@@ -48,6 +48,9 @@ namespace hpp {
       ConfigurationShooterBuilder_t;
     typedef boost::function <SteeringMethodPtr_t (const ProblemPtr_t&) >
       SteeringMethodBuilder_t;
+		typedef std::vector<boost::shared_ptr<model::CollisionObject> > 
```

Comment:
Why not using typedef CollisionObjectShPtr for boost::shared_ptr<model::CollisionObject> ?